src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringService.kt

```diff
@@ -294,8 +374,8 @@ class JournalføringService(
         taskService.save(OpprettOppgaveForOpprettetBehandlingTask.opprettTask(opprettOppgaveTaskData))
     }
 
-    private fun ferdigstillJournalføringsoppgave(journalføringRequest: JournalføringRequest) {
-        oppgaveService.ferdigstillOppgave(journalføringRequest.oppgaveId.toLong())
+    private fun ferdigstillJournalføringsoppgave(oppgaveId: Long) {
```

Comment:
unødvendig metode

Reply:
Den er i bruk 4 steder, så litt usikker på hvorfor den er unødvendig? 🤔

Reply:
Kan bare gjøre det inline, tenker du? 
Forsåvidt enig, men noe med lesbarheten på "alt som skjer" jeg er bekymret for...